odoo/addons/base/models/ir_mail_server.py

```diff
@@ -112,6 +113,12 @@ class IrMailServer(models.Model):
                                                                   "is used. Default priority is 10 (smaller number = higher priority)")
     active = fields.Boolean(default=True)
 
+    from_filter = fields.Char(
```

Comment:
HumanLinter: no need of void line between fields + maybe put it above smtp_host. We could reorder fields like
```
# description
name
active
# usage
sequence
from_filter
# smtp
...
```